README.md

```diff
@@ -1,12 +1,40 @@
-# Template project
-
-> Template for [Axone](https://axone.xyz) opensource projects.
+# Hub backend
 
 [![lint](https://img.shields.io/github/actions/workflow/status/axone-protocol/template-oss/lint.yml?branch=main&label=lint&style=for-the-badge&logo=github)](https://github.com/axone-protocol/template-oss/actions/workflows/lint.yml)
```

Comment:
By the way, you could take the opportunity of this PR to add the status badge for the build.